bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -241,13 +240,21 @@ static int callAndWait(String[] pstr, ToIntFunction<IUnknown> callable) {
 	pstr[0] = null;
 	phr[0] = callable.applyAsInt(completion);
 	completion.Release();
-	Display display = Display.getCurrent();
 	while (phr[0] == COM.S_OK && pstr[0] == null) {
-		if (!display.readAndDispatch()) display.sleep();
+		processNextOSMessage();
 	}
 	return phr[0];
 }
 
+private static void processNextOSMessage() {
```

Comment:
Would be great to have a short comment explaining what the method is doing or why it is needed for Edge.